src/pages/instances/InstanceOverviewDevices.tsx

```diff
@@ -0,0 +1,67 @@
+import { FC } from "react";
+import { MainTable } from "@canonical/react-components";
+import { LxdInstance } from "types/instance";
+import {
+  LxdGPUDevice,
+  LxdNoneDevice,
+  LxdOtherDevice,
+  LxdProxyDevice,
+} from "types/device";
+
+interface Props {
+  instance: LxdInstance;
+}
+type InstanceOverviewDevices =
+  | LxdNoneDevice
+  | LxdGPUDevice
+  | LxdOtherDevice
+  | LxdProxyDevice;
+
+const InstanceOverviewDevices: FC<Props> = ({ instance }) => {
+  const instanceDevices = Object.entries(
+    instance.expanded_devices ?? {},
+  ).filter(([, device]) => {
+    return !(device.type === "nic" || device.type === "disk");
+  });
+
+  const hasDevices = instanceDevices.length > 0;
+
+  const deviceHeaders = [
+    { content: "Name", sortKey: "name", className: "u-text--muted" },
+    { content: "Type", sortKey: "type", className: "u-text--muted" },
+  ];
+
+  const deviceRows = instanceDevices.map(([devicename, dev]) => {
+    const device = dev as InstanceOverviewDevices;
+
+    return {
+      columns: [
+        {
+          content: devicename,
+          role: "rowheader",
+          "aria-label": "Name",
+        },
+        {
+          content: device.type,
+          role: "rowheader",
+          "aria-label": "Type",
+        },
+      ],
+      sortData: {
+        name: device.name?.toLowerCase(),
+        type: device.type,
+      },
+    };
+  });
+
+  return (
+    <>
+      {hasDevices && (
+        <MainTable headers={deviceHeaders} rows={deviceRows} sortable />
+      )}
+      {!hasDevices && <>No devices available</>}
```

Comment:
```suggestion
      {!hasDevices && <>-</>}
```
I think this is too verbose, we can have the same info with much less copy. This is also better in line how we deal with no networks.